nvflare/private/fed/server/message_send.py

```diff
@@ -74,6 +74,7 @@ def send_requests(
         if job_id:
             fqcn = FQCN.join([client.name, job_id])
             channel = CellChannel.CLIENT_COMMAND
+            optional = True
```

Comment:
Why is this optional?
I think we have messages sent via this channel that are required?

Reply:
Because all messages for CLIENT COMMAND channel should be optional to suppress unnecessary CELL ERROR messages.